packages/land/contracts/Land.sol

```diff
@@ -153,6 +164,18 @@ contract Land is LandBaseToken, OperatorFiltererUpgradeable {
         return royaltyManager;
     }
 
+    /// @notice Get the address of the owner
+    /// @return ownerAddress The address of the owner.
+    function owner() external view returns (address ownerAddress) {
+        return _owner;
+    }
+
+    /// @notice Set the address of the new owner of the contract
+    /// @param _newOwner address of new owner
+    function transferOwnership(address _newOwner) external onlyAdmin {
```

Comment:
check for address(0) ?

Reply:
[ERC173](https://eips.ethereum.org/EIPS/eip-173#specification) says `To renounce any ownership of a contract set _newOwner to the zero address: transferOwnership(address(0)). If this is done then a contract is no longer owned by anybody.`
We can add check as required.